### PR TITLE
checker: Move `linecount` line-by-line feedback to `extended_feedback`

### LINKED ISSUE #407: Feedback from `linecount` checker overwrites IR feedback
After #398, we run `linecount` even if the submission gets an `IR` verdict. This results in the per-case feedback overwriting the error message's feedback.

Possible solutions include
1. Moving the per-case feedback into `extended_feedback`
2. Creating a new field to handle RTE/IR feedback

### PR DESCRIPTION
**Pull request by kiritofeng**
Closes #407

### CODE REVIEW

**Comment by quantum5** [comment]
Can you fix the spelling error in the commit message?

File: dmoj/checkers/linecount.py [hunk 1]
```diff
@@ -28,7 +28,8 @@ def check(
             count += 1
 
     return CheckerResult(
-        count == len(judge_lines), point_value * (1.0 * count / len(judge_lines)), ''.join(cases) if feedback else ""
+        count == len(judge_lines), point_value * (1.0 * count / len(judge_lines)),
+        extended_feedback='Case Feedback:\n' + '\n'.join(cases) if feedback else ''
```
Comment (quantum5) [inline dmoj/checkers/linecount.py:32]: `''.join` might be better so it's not one giant column going all the way down.

@Xyene any thoughts?

Comment (fluix-dev) [reply dmoj/checkers/linecount.py:32]: Not Xyene, but if my memory serves me right then `''.join` is what MCPT does and it looks quite nice. I think the giant column wouldn't look nice either.

**Review by quantum5 — APPROVED** [review]

**Review by Xyene — APPROVED** [review]
Thanks!

**Pull request merged**